openwisp_monitoring/check/migrations/0003_create_ping.py

```diff
@@ -0,0 +1,26 @@
+from django.db import migrations
+from openwisp_monitoring.check.settings import AUTO_PING
+from openwisp_monitoring.check.tasks import auto_create_ping
+
+
+def create_device_ping(apps, schema_editor):
+    if AUTO_PING:
+        Device = apps.get_model('config', 'Device')
+        for device in Device.objects.all():
+            auto_create_ping(model=Device.__name__.lower(),
+                            app_label=Device._meta.app_label,
```

Comment:
indentation is off by one space, but let's not worry about this anymore since it will be handled by the black formatter soon.